db/migrations/sqlite/000065_pin_signer.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pins ADD COLUMN signer TEXT;
+UPDATE pins SET signer = "";
```

Comment:
This was something significant I found as I was verifying the code against this table.

If the `pins` doesn't have the signing key reference in there, then for private messages we cannot verify the signing key against the `key` field in the message.

I did consider avoiding the full Eth address / Fabric DN string in the pins table - but a join table seemed like a lot of complexity for an unknown optimization benefit.

![](https://user-images.githubusercontent.com/6660217/132556591-f454d893-3812-4d19-b052-1d38365d4602.png)